ui/job-view/pushes/PushHeader.jsx

```diff
@@ -132,14 +132,13 @@ export default class PushHeader extends React.PureComponent {
   cancelAllJobs() {
     if (window.confirm('This will cancel all pending and running jobs for this push. It cannot be undone! Are you sure?')) {
       const { push, isLoggedIn } = this.props;
-      const jobsCanCancel = push.jobList
-        .filter(({ state }) => state === 'running' || state === 'pending')
-        .map(({ id }) => id);
+      // Any job Id inside the push will do
+      const jobId = push.jobList[0].id;
```

Comment:
In fact, no job is necessary :)

Reply:
Ah, OK -- then this should stay, but don't pass a taskId to `TaskclusterModel.submit`